who-metrics-ui/src/data/data.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Should we add this file to the `.gitignore`? Since it's generated in the `nextjs` action, I wouldn't think that it would need to be pushed to the repo manually by users.

Reply:
Yeah, I initially left this because this file will need to be present for the frontend import to work correctly in the future. But I think it's better to update the frontend instructions to make sure to generate this file first.

Reply:
Updated in 8a7875d42eb35a7a6660ba61db75987ac51ee0a6 - will update the instructions once this becomes a dependency of the frontend, too!